helm-chart/kubray-operator/values.yaml

```diff
@@ -5,8 +5,8 @@
 replicaCount: 1
 
 image:
-  repository: docker.io/library/controller
-  tag: latest
+  repository: kuberay/operator
```

Comment:
Maybe `rayproject/kuberay-operator` is the correct one?

Reply:
`rayproject/kuberay-operator` is the mirror of `kuberay/operator`, for the image management, please check https://github.com/ray-project/ray/pull/21086#discussion_r788291269 for details. 

`rayproject/kuberay-operator` is recently sync to `rayproject` namespace for autoscalign project. We will need a better way to sync the images in the future

Reply:
I would suggest to use `kuberay/operator` at this moment.